protocol-units/settlement/mcr/client/src/stub.rs

```diff
@@ -104,35 +103,35 @@ pub mod test {
 		let guard = client.commitments.write().await;
 		assert_eq!(guard.get(&1), Some(&commitment));
 
-        assert_eq!(*client.current_height.read().await, 1);
-        assert_eq!(client.get_max_tolerable_block_height().await?, 17);
+		assert_eq!(*client.current_height.read().await, 1);
+		assert_eq!(client.get_max_tolerable_block_height().await?, 17);
 
 		Ok(())
 	}
 
-    #[tokio::test]
-    async fn test_post_block_commitment_batch() -> Result<(), anyhow::Error> {
-        let client = McrSettlementClient::new();
-        let commitment = BlockCommitment {
-            height: 1,
-            block_id: Default::default(),
-            commitment: Commitment::test(),
-        };
-        let commitment2 = BlockCommitment {
-            height: 2,
-            block_id: Default::default(),
-            commitment: Commitment::test(),
-        };
-        client.post_block_commitment_batch(vec![
-            commitment.clone(),
-            commitment2.clone(),
-        ]).await.unwrap();
-        let guard = client.commitments.write().await;
-        assert_eq!(guard.get(&1), Some(&commitment));
-        assert_eq!(guard.get(&2), Some(&commitment2));
-        Ok(())
-    }
-	
+	#[tokio::test]
+	async fn test_post_block_commitment_batch() -> Result<(), anyhow::Error> {
```

Comment:
It will be nice to have antagonistic tests at some point, though this may only be reasonable with the a stub contract implemented.

Reply:
I test the commitment batch with the other calls in one unit test because all the smart contract call must be done in the right order.